content/en/user-guide/chaos-engineering/web-application-dashboard/index.md

```diff
@@ -1,20 +1,20 @@
 ---
 title: "Chaos Engineering Dashboard"
 linkTitle: "Chaos Engineering Dashboard"
-weight: 2
-description: Effortlessly design, activate, and manage fault injection experiments with the LocalStack user-friendly dashboard.
+description: Effortlessly design, activate, and manage Fault Injection Service experiments
```

Comment:
once we update the web app to the chaos api, we'd need to update this here as well.